clang/lib/Driver/Driver.cpp

```diff
@@ -1239,8 +1263,17 @@ void Driver::CreateOffloadingDeviceToolChains(Compilation &C,
         << ArgValue << A->getOption().getName();
   };
 
+  // TODO: Transition to using -fsycl-link as a flag as opposed to an option
+  // that takes an argument.  The use of 'default' is a temporary solution as we
+  // remove FPGA support.
   Arg *SYCLLink = getArgRequiringSYCLRuntime(options::OPT_fsycl_link_EQ);
-  checkSingleArgValidity(SYCLLink, {"early", "image"});
+  checkSingleArgValidity(SYCLLink, {"early", "image", "default"});
```

Comment:
If use of `-fsycl-link=early` and `-fsycl-link=image` are not supported, do we still need to pass "`early`" and "`image`" as `AllowedValues` to `checkSingleArgValidity` ?

Reply:
No - this isn't necessary.  The purpose of this change was to introduce the functionality to reject the options.  Cleanup will occur later.